whc-shared/src/main/java/com/ibm/whc/deid/shared/util/MaskingConfigUtils.java

```diff
@@ -396,18 +225,19 @@ protected void validateRules(DeidMaskingConfig deidMaskingConfig)
 
         if (!ruleNames.add(ruleName)) {
           throw new InvalidMaskingConfigurationException(
-              "invalid masking configuration: the value of the `" + Rule.NAME_PROPERTY_NAME
-                  + "` property in the rule at offset " + offset + " in `"
+              "invalid masking configuration: the value `" + ruleName + "` is used for the `"
```

Comment:
We should try to use the WPHnnnnX codes as much as possible in case we ever want to 1) document known errors and solutions (the code is useful as a reference) or 2) translate the errors one day especially when they get returned in the UI. 
Seems like the train already left the station and there's error message strings everywhere instead of the logCodes.properties